card/mod/standard/set/all/rich_html/menu.rb

```diff
@@ -22,7 +22,8 @@
   end
 
   def menu_path_opts
-    opts = { slot: { home_view: voo.home_view } }
+    opts = { slot: { home_view: (voo.home_view || @slot_view),
```

Comment:
OK, then I probably won't touch this for now.

I'm thinking in the future we should write some custom (non-slot) javascript for the menu so that links within the main menu can remain simple slotter links and retain all the power that entails.  

but I'll merge this now and move on.